addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -158,12 +158,10 @@ def _generate_efaktur_invoice(self, delimiter):
 
             street = ', '.join([x for x in (move.partner_id.street, move.partner_id.street2) if x])
 
-            invoice_npwp = '000000000000000'
-            if commercial_partner.vat and len(commercial_partner.vat) >= 12:
-                invoice_npwp = commercial_partner.vat
-            elif (not commercial_partner.vat or len(commercial_partner.vat) < 12) and commercial_partner.l10n_id_nik:
-                invoice_npwp = commercial_partner.l10n_id_nik
-            invoice_npwp = invoice_npwp.replace('.', '').replace('-', '')
+            invoice_npwp = move.partner_id.vat.replace('.', '').replace('-', '')
+            etax_name = move.partner_id.name
+            if invoice_npwp[:15] == '000000000000000' and move.partner_id.l10n_id_nik:
+                etax_name = "%s#NIK#NAMA#%s" % (move.partner_id.l10n_id_nik, etax_name)
```

Comment:
```suggestion
            invoice_npwp = commercial_partner.vat.replace('.', '').replace('-', '')
            etax_name = move.partner_id.name
            if invoice_npwp[:15] == '000000000000000' and commercial_partner.l10n_id_nik:
                etax_name = "%s#NIK#NAMA#%s" % (commercial_partner.l10n_id_nik, etax_name)
```
Do we no longer use the l10n_id_tax_name field?